src/preferences/dialog/dlgprefbroadcast.cpp

```diff
@@ -126,9 +126,9 @@ DlgPrefBroadcast::DlgPrefBroadcast(QWidget *parent, UserSettingsPointer _config)
     comboBoxEncodingFormat->setCurrentIndex(tmp_index);
 
     // Encoding channels combobox
-    comboBoxEncodingChannels->addItem(tr("Stereo"), BROADCAST_CHANNELS_STEREO);
+    comboBoxEncodingChannels->addItem(tr("Automatic"), BROADCAST_CHANNELS_AUTOMATIC);
```

Comment:
Well, It was as useless before. I just changed the description given the change to mono for lower bitrates (and only on MP3).
Anyway, an option could be added to force stereo or mono instead of this automatic change. But then it would need to be done for Vorbis too.